modules/theme-tools/responsive-videos.php

```diff
@@ -42,3 +43,51 @@ function jetpack_responsive_videos_embed_html( $html ) {
 
 	return '<div class="jetpack-video-wrapper">' . $html . '</div>';
 }
+
+/**
+ * Check if oEmbed is YouTube or Vimeo before wrapping.
+ *
+ * @return string
+ */
+function jetpack_responsive_videos_maybe_wrap_oembed( $html, $url ) {
+	if ( empty( $html ) || ! is_string( $html ) || ! $url ) {
+		return $html;
+	}
+
+	$is_video = false;
```

Comment:
I don't think this is necessary.  If the preg_match fails below, won't it return false anyway?